pygls/protocol.py

```diff
@@ -683,22 +679,34 @@ def get_configuration(self, params, callback):
         """
         return self.send_request(WORKSPACE_CONFIGURATION, params, callback)
 
-    def get_configuration_async(self, params):
+    def get_configuration_async(self, params: ConfigurationParams) -> asyncio.Future:
         """Calls `get_configuration` method but designed to use with coroutines
 
         Args:
-            params(dict): ConfigurationParams from lsp specs
+            params(ConfigurationParams): ConfigurationParams from lsp specs
         Returns:
             asyncio.Future that can be awaited
         """
-        return asyncio.wrap_future(self.get_configuration(params, None))
+        return asyncio.wrap_future(self.get_configuration(params))
+
+    def log_trace(self, message: str, verbose: Optional[str] = None) -> None:
+        """Sends trace notification to the client."""
+        if self.trace == Trace.Off:
+            return
+
+        params = LogTraceParams(message=message)
+        if self.trace == Trace.Verbose and verbose:
```

Comment:
Very minor thing - to me it's a lot more intuitive to check `verbose` first and then if equality